lib/stats/aggregator/stats-change-per-second.c

```diff
@@ -239,7 +240,7 @@ _is_less_then_duration(StatsAggregatorCPS *self, CPSLogic *logic, time_t *now)
   if (logic->duration == -1)
     return TRUE;
 
-  return _calc_sec_between_time(&self->init_time, now) <= logic->duration;
+  return (gsize)_calc_sec_between_time(&self->init_time, now) <= logic->duration;
```

Comment:
You still seem to have a couple of gsize casts here and there.

Reply:
Updated 👍